cvpy/image/Image.py

```diff
@@ -0,0 +1,268 @@
+#!/usr/bin/env python
+# encoding: utf-8
+#
+# Copyright SAS Institute
+#
+#  Licensed under the Apache License, Version 2.0 (the License);
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+''' Image analysis tools '''
+
+import sys
+import struct
+import numpy as np
+from typing import List
+from warnings import warn
+from swat import CAS, CASTable
+
+class Image(object):
```

Comment:
All of the methods in this class are static. So similar to __reverse, the remaining methods can also be annotated as staticmethod.

Reply:
Thanks, Pankaj! I made this change for each of the functions. Let me know if you have any other comments.